.github/workflows/ci.yml

```diff
@@ -223,24 +241,107 @@ jobs:
           targets: wasm32-unknown-unknown
       - uses: Swatinem/rust-cache@v1
       - run: cargo install sqlx-cli
-
       - name: Install wasm-snip
         run: cargo install wasm-snip
-
       - name: Build fuel-indexer-test WASM
-        run: |
-          cargo build -p fuel-indexer-test --release --target wasm32-unknown-unknown
+        run: >
+          cargo build -p fuel-indexer-test --release --target
+          wasm32-unknown-unknown
+
           bash scripts/stripper.bash fuel_indexer_test.wasm
-          cp fuel_indexer_test.wasm target/wasm32-unknown-unknown/release/
 
+          cp fuel_indexer_test.wasm target/wasm32-unknown-unknown/release/
       - name: Build simple-wasm WASM
         run: |
           cargo build -p simple-wasm --release --target wasm32-unknown-unknown
           bash scripts/stripper.bash simple_wasm.wasm
           cp simple_wasm.wasm target/wasm32-unknown-unknown/release/
-
       - name: Run E2E and integration tests
-        run: cargo test --locked --all-targets --features e2e,postgres -p fuel-indexer-tests -- --test-threads=1
+        run: >-
+          cargo test --locked --all-targets --features e2e,postgres -p
```

Comment:
261-304 is the only thing I added to CI - the rest is just formatting from the prettier